homeassistant/components/amberelectric/sensor.py

```diff
@@ -160,6 +160,7 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
             datum["end_time"] = interval.end_time.isoformat()
             datum["renewables"] = round(interval.renewables)
             datum["spike_status"] = interval.spike_status.value
+            datum["descriptor"] = normalize_descriptor(interval.descriptor)
```

Comment:
Can we remove this too?

Reply:
No. That is the forecast sensor. There is no equivalent in the price descriptor sensor.

Reply:
Ok.